flag-challenge/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Flags</title>
+  </head>
+  <body>
+    <section>
+      <div id="flag_stripe_one"></div>
```

Comment:
just remember the naming conventions is 2 underscores example __

Reply:
I literally forget this every time haha thank you!